app/frontend/components/common/contactForm.js

```diff
@@ -0,0 +1,104 @@
+const {h, Component} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../actions')
+const Modal = require('./modal')
+const {ADALITE_CONFIG} = require('../../config')
+
+class ContactForm extends Component {
+  constructor(props) {
+    super(props)
+    this.closeContactFormModal = this.closeContactFormModal.bind(this)
+  }
+
+  closeContactFormModal() {
+    this.props.closeContactFormModal()
+  }
+
+  render() {
+    return h(
+      Modal,
+      {},
+      h(
+        'main',
+        {class: 'contact-modal'},
+        h('h2', {class: 'export-title'}, 'Contact Form'),
+        h('div', {}, 'consider the following:'),
+        h(
+          'ul',
+          {class: 'instructions'},
+          h('li', {}, 'have you tried disabling your ad blocker?'),
+          h(
+            'li',
+            {},
+            'if you are using a hw wallet, is its firmware up to date? does it work on myetherwallet?'
+          ),
+          h('li', {}, 'what is your browser and operating system?')
+        ),
+        h(
+          'form',
+          {
+            class: 'contact-form',
+            id: 'contactForm',
+            method: 'POST',
+            action: `//formspree.io/${ADALITE_CONFIG.ADALITE_SUPPORT_MAIL}`,
+          },
+          h(
+            'div',
+            {
+              class: 'form-row',
+            },
+            h('input', {
+              type: 'text',
+              autocomplete: 'off',
+              placeholder: 'Your name',
+              name: 'name',
+              class: 'input fullwidth',
+            }),
+            h('input', {
+              type: 'email',
+              autocomplete: 'off',
+              placeholder: 'Your email',
+              name: '_replyto',
+              class: 'input fullwidth',
+            })
+          ),
+          h('textarea', {
+            placeholder: 'Your issue',
+            autocomplete: 'off',
+            name: 'message',
+            class: 'input fullwidth textarea',
+          }),
+          h('input', {
+            type: 'hidden',
+            name: '_subject',
+            value: 'Support Contact Form',
+          }),
+          h('input', {
+            type: 'text',
+            name: '_gotcha',
+            style: 'display:none',
+          }),
+          h('br', {}),
```

Comment:
not sure if this is clean. Shouldn't the spacing rather be achieved with an appropriate padding?

Reply:
`br` removed.